simphony/cuds/abstractlattice.py

```diff
@@ -0,0 +1,77 @@
+# -*- coding: utf-8 -*-
+"""
+    Module for Abstract Lattice class:
+        ABCLattice ---> Common Base abstract class ("interface") for
+            the Lattice.
+"""
+
+from __future__ import print_function
+from abc import ABCMeta, abstractmethod
+
+
+class ABCLattice(object):
+    """Abstract base class for a Lattice item."""
```

Comment:
Should the `name` attribute be added to the docstring?

How about the other attributes used by Lattice ( `type, base_vect,  size, origin`)?  I assume they will be supported by FileLattice and any other classes implementing the ABCLattice interface.